src/styles/header.scss

```diff
@@ -0,0 +1,113 @@
+header {
+    position: fixed;
+    z-index: 999;
+    width: 100%;
+    height: 3rem;
+    min-height: 50px;
+    color: #fff;
```

Comment:
White is used as colour of the font a lot

worth making some Theme constants file to easily reuse and if needed change basic style of the page